quilt/tools/command.py

```diff
@@ -648,6 +654,19 @@ def install(package, hash=None, version=None, tag=None, force=False):
             )
         )
         pkghash = response.json()['hash']
+    elif len(hash) < 64:
+        assert len(hash) >= 6   # short is ok, but only to a point...
```

Comment:
This can come from the command-line args. An error is ok - but make sure it doesn't assert on user input.

Reply:
DONE